Triggers/RemoveLightSourcesTrigger.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Celeste.Mod.Entities;
+using Monocle;
+using Microsoft.Xna.Framework;
+using System.Collections.Generic;
+
+namespace Celeste.Mod.SpringCollab2020.Triggers {
+    [CustomEntity("SpringCollab2020/RemoveLightSourcesTrigger")]
+    [Tracked]
+    class RemoveLightSourcesTrigger : Trigger {
+        public RemoveLightSourcesTrigger(EntityData data, Vector2 offset) : base(data, offset) {
+            IsPersistent = data.Bool("isPersistent", true);
+            level = SceneAs<Level>();
+        }
+
+        public static void Load() {
+            Everest.Events.Level.OnLoadLevel += LevelLoadHandler;
+            Everest.Events.Level.OnExit += OnExitHandler;
+        }
+
+        public static void Unload() {
+            Everest.Events.Level.OnLoadLevel -= LevelLoadHandler;
+            Everest.Events.Level.OnExit -= OnExitHandler;
+        }
+
+        private static void LevelLoadHandler(Level loadedLevel, Player.IntroTypes playerIntro, bool isFromLoader) {
+            if(loadedLevel.Session.GetFlag("lightsDisabled") == true) {
```

Comment:
How did I miss this? O.o

```suggestion
            if (loadedLevel.Session.GetFlag("lightsDisabled")) {
```